Scripts/application.py

```diff
@@ -114,83 +118,85 @@ def return_person(person_id: str) -> str:
+@app.route('/api/v1/forbidden_pairing')
+def get_forbidden_pairings():
     
-        # verify that request contains a valid request  
-        try:
-            request_json = request.get_json()
-        except:
-            abort(
-                400,
-                description = "Please verify that request contains valid json.")
-        try:
-            new_giver_constraint = request_json['santa_ID']
-        except:
-            abort(
-                400,
-                description = "Please verify that request contains a "
-                              "'santa_ID' key.")
-        try:
-            new_receiver_constraint = request_json['assignment_ID']
-        except:
-            abort(
-                400,
-                description = "Please verify that request contains an "
-                              "'assignment_ID' key.")
+    # returns a json of all forbidden pairings
+    with sqlite3.connect(path_to_db) as connection:
+        cursor = connection.cursor()
+        cursor.execute('SELECT constraint_ID, santa, santa_ID, assignment, '
+                       'assignment_ID FROM forbiddenPairings;')
+        raw_data: List = cursor.fetchall()
+        data: List = []
+        for pair in raw_data:
+            data.append(
+                {
+                    "constraint_ID": pair[0],
+                    "santa": pair[1],
+                    "santa_ID": pair[2],
+                    "assignment": pair[3],
+                    "assignment_ID": pair[4]})
+        json_data = json.dumps(data)
+    return json_data
         
-        # add new constraint to database
-        with sqlite3.connect(path_to_db) as connection:
-            cursor = connection.cursor()
-            command = 'SELECT (email) FROM people WHERE person_ID = ?;'
-            cursor.execute(command, (new_giver_constraint,))
-            giver_ID = cursor.fetchone()[0]
-            cursor.execute(command, (new_receiver_constraint,)) # finish swapping IDs for emails!!!
-            receiver_ID = cursor.fetchone()[0]
-            command = ('INSERT into forbiddenPairings (santa, santa_ID '
-                       'assignment, assignment_ID) VALUES (?, ?, ?, ?);')
-            cursor.execute(
-                command,
-                (
-                    new_giver_constraint,
-                    giver_ID,
-                    new_receiver_constraint,
-                    receiver_ID))
-             
-            #return posted record
-            cursor.execute('SELECT last_insert_rowid()')
-            posted_record: List = cursor.fetchone()
-            data: Dict = {
-                "constraint_ID": posted_record[0],
-                "santa": posted_record[1],
-                "santa_ID": posted_record[2], 
-                "assignment": posted_record[3],
-                "assignment_ID": posted_record[4]}
-            return json.dumps(data)
+@app.route('/api/v1/forbidden_pairing', methods = ["POST"])
+def forbidden_pairing_request():

```

Comment:
as requested please adopt the convention of [HTTP_method]_forbidden_pairing() for each HTTP method

Reply:
This has been included with the latest commit.